nexis/lib/pages/auth/auth_page.dart

```diff
@@ -53,68 +56,93 @@ class AuthPageState extends State<AuthPage> {
 
   @override
   Widget build(BuildContext context) {
+    var mediaQuery = MediaQuery.of(context);
+    var screenType = getScreenType(mediaQuery);
+
+    switch (screenType) {
+      case ScreenType.mobile:
+        return AuthPage(context);
+      case ScreenType.tablet:
+        return AuthPage(context);
+      case ScreenType.desktop:
+        return AuthPage(context);
+      default:
+        return AuthPage(context);
+    }
```

Comment:
Remove this and build the AuthPage directly without the screenType since you are just building the same widgit for every window size, this just does too many unnecessary rebuilds.

`
@override
  Widget build(BuildContext context) {
      return AuthPage(context);
  }
  `